src/raiden_libs/messages/balance_proof.py

```diff
@@ -1,3 +1,5 @@
+from typing import Any, Type
```

Comment:
With the `libs.messages.monitor_request` module removed, is this still needed or can we remove it?

Reply:
Clirification: I'm talking about the whole `Message` and `BalanceProof` classes.

Reply:
It's still used at least in `test_e2e` via `get_balance_proof` as well as in many raiden_libs tests. We can probably remove it with a few changes and lots of deleted raiden_libs code. But let's do that in a different PR.